test/test_core.py

```diff
@@ -6014,6 +6014,10 @@ def test_fs_open_wasmfs(self):
     self.emcc_args += ['-sFORCE_FILESYSTEM']
     self.do_runf(test_file('fs/test_open_wasmfs.c'), 'success')
 
+  def test_fs_close(self):
+    self.set_setting("FORCE_FILESYSTEM")
+    self.do_runf(test_file('fs/test_close.c'), 'success')
```

Comment:
Again, I wonder if this is tests elsewhere already.. and if not, perhaps we should add a more general "test_fs_js_api" test to catch places there its not otherwise tested?

Reply:
Just to clarify, what do you mean catch places? Would this be different than simply consolidating all of the JS API tests into a "test_fs_js_api" file?

Reply:
I guess I'm not sure we need individual tests for the JS API.. or if we do we should mark them as such (e.g. `test_fs_jsapi_close`)..  the naming of this test makes me think its testing the C `close()` function which I would imagine we already have tests for elsewhere.

Reply:
@sbc100 see my comment above - this adds stuff not tested anywhere else, since we don't test the return code of `FS.close` anywhere until this PR.

But I like the idea to add a single new `test_fs_js_api` for stuff like this that is purely for the JS API. There is no need to have many small tests for such things. So we can rename the new test here to that, I think, and then keep adding to it, how does that sound?

Reply:
Oh, ignore the first part of my response - since the old FS didn't return a value, and threw, then there was no return value to test... :smile: 

Regardless I think a single new test for the JS FS API makes sense.